packages/core/src/indexing/types.ts

```diff
@@ -0,0 +1,66 @@
+import type { StreamID } from '@ceramicnetwork/streamid'
+import type { DID } from 'dids'
+
+export interface IndexStreamArgs {
+  streamID: StreamID
+  model: StreamID
+  controller: DID
+  lastAnchor: Date | null
+  content: Record<string, any>
+  relationFields?: Map<string, StreamID | DID>
+}
+
+export enum Ordering {
```

Comment:
I thought the conclusion on the spec was to leave Ordering out entirely for now

Reply:
We better have it here.

If we go with a single ordering method while implementing the indexer, there is a high chance for it to be omitted in some internal API. The consequence is at the later stage, when we really need multiple methods, we would need to refactor some of the code. That will happen at an unfortunate moment when most of the code is solidified. To me it makes more sense to approach indexing code now having in mind and at hand variability of ordering methods, even if only one of the options is finally exposed to a user.

Reply:
okay, so long as it doesn't wind up exposed to any user-facing APIs, and we can change the names used later if we want to do before exposing them, then that's fine